### PR TITLE
Allows instanciating custom Cesium imageries

### PR DESCRIPTION
**Pull request by gberaudo**
This requires compiling with Closure.

### CODE REVIEW

**Comment by gberaudo** [comment]
Any review?

**Comment by fredj** [comment]
LGTM

File: src/rastersynchronizer.js [hunk 1]
```diff
@@ -62,15 +62,27 @@ olcs.RasterSynchronizer.prototype.removeAllCesiumObjects = function(destroy) {
...
+ * Creates a Cesium.ImageryLayer.
+ * May be overriden by child classes to implement custom behavior.
+ * The default implementation handles tiled imageries in EPSG:4326 or
+ * EPSG:3859.
+ * @param {!ol.layer.Layer} olLayer
+ * @param {?ol.proj.Projection} viewProj Projection of the view.
+ * @return {?Cesium.ImageryLayer} null if not possible (or supported)
+ * @protected
+ */
+olcs.RasterSynchronizer.prototype.convertLayerToCesiumImagery =
```
Comment (fredj) [inline src/rastersynchronizer.js]: could be:
```
olcs.RasterSynchronizer.prototype.convertLayerToCesiumImagery = olcs.core.tileLayerToImageryLayer
```

**Comment by gberaudo** [comment]
Thank you for the review @fredj, I addressed your comment and will now merge.

**Pull request merged**